docs/core/prerequisites.md

```diff
@@ -0,0 +1,463 @@
+---
+title: Prerequisites for .NET Core on Windows, Linux, MacOS
+description: Supported Windows, Linux, MacOS versions and .NET Core dependencies to develop, deploy, and run .NET Core applications.
+author: leecow
+ms.author: leecow
+ms.date: 10/30/2019
+zone_pivot_groups: operating-systems-set-one
+---
+
+# Prerequisites for .NET Core
+
+This article provides information on getting started with .NET Core. .NET Core can be installed on Windows, Linux, and macOS. You can code in your favorite text editor and produce cross-platform libraries and applications.
+
+.NET Core is made up of two components, the runtime and the SDK. The **runtime** is required to run a .NET Core application that doesn't include the runtime. The **SDK** is required to build .NET Core applications.
+
+There are different kinds of ways to develop .NET Core apps:
+
+::: zone pivot="os-windows"
```

Comment:
Wondering what this line is ?
Does it render in some special way ?

Reply:
Only appears when the appropriate option is chosen at the top of the article:

![image](https://user-images.githubusercontent.com/2672110/67890224-8c63ad00-fb0d-11e9-9581-bf6b7dc02c1c.png)

Reply:
@Thraka, Didn't know about it. Is it documented somewhere ?
It's not explained in the template.md file.

Reply:
It's an unused feature really. We're not allowed to use it unless in an approved circumstance. This is a bit of a pilot to see how it is received.